src/sass/_common/common.scss

```
@@ -12,7 +12,7 @@ body {
     background: repeat-x bottom;
     overflow-x: hidden;
     font-family: $FONT-STACK !important;
-    text-rendering: optimizeLegibility;
+    text-rendering: optimizelegibility;
```

as far as checked 
```suggestion
    text-rendering: optimizeLegibility;
```
this one is the correct one.

---

unfortunately, with stylelint version 15, the correct one is `optimizelegibility` :)
>"Running "stylelint:all" (stylelint) task
>
>  src/sass/_common/common.scss:15:21
>  ✖  15:21  Expected optimizeLegibility to be optimizelegibility"